### DISCUSSION TITLE
winodws7 SpeechSynthesisUtterance libcef using web api Dll error

### DISCUSSION THREAD

**Discussion by ArmyMedalMei** [cefsharp/CefSharp#4247] [General]
Html Javascript Code :

utter = new window.SpeechSynthesisUtterance("cat");
window.speechSynthesis.cancel();
window.speechSynthesis.speak(utter);

In the win7 environment, libcef.dll will report an error, but it will sound normally in the win10 environment

- **What architecture x86 or x64?**
x64
	
- **What version of .Net?**
    .Net 4.7.2
    
- **On what operating system?**
    Win7

- **Are you using `WinForms`, `WPF` or `OffScreen`?**
    WinForms
    
- **What steps will reproduce the problem?**

![image](https://user-images.githubusercontent.com/19337530/190841610-0ac40239-bc6c-4963-9c15-c88c5c5b6d4c.png)

**Comment by ArmyMedalMei** [comment]
win7 Ultimate This error can occur

**Reply by amaitland** [reply]
Converted to discussion as this is not something that can be fixed in CefSharp directly. The Speech API is implement entirely at the Chromium level.

You also have skipped large part of the bug report, you haven't done any of the testing required. 

Have you confirmed chrome at the same version works in Windows 7?

> the win7 environment, libcef.dll will report an error, but it will sound normally in the win10 environment

As per the bug report temple. 

If you are seeing a crash in libcef.dll then please download libcef.dll.pdb and place it next to libcef.dll to obtain a detailed stack trace, see https://github.com/cefsharp/CefSharp/wiki/Trouble-Shooting#loading-native-symbols-for-easier-diagnosis for details.

Does the cef log provide any relevant information? (By default there should be a debug.log file in your bin directory)

> Does the problem reproduce using the [MinimalExample](https://github.com/cefsharp/CefSharp.MinimalExample)?

Please test with the minimal example and confirm the issue reproduces. 

> Does this problem also occur in the CEF Sample Application?

You'll also need to test with the CEF Sample application as detailed in the bug report temple. Complete the testing as per the bug report temple. 

https://github.com/cefsharp/CefSharp/blob/master/.github/ISSUE_TEMPLATE/bug_report.md#bug-report
